tests/nnstreamer_datarepo/datareposrc/unittest_datareposrc.cc

```diff
@@ -0,0 +1,31 @@
+#include <gtest/gtest.h>
+#include <glib.h>
+#include <glib/gstdio.h>
+#include <gst/gst.h>
+#include <unittest_util.h>
+
+/**
+ * @brief Main GTest
+ */
+int
+main (int argc, char **argv)
+{
+  int result = -1;
+  g_warning ("datareposrc test");
+  try {
+    testing::InitGoogleTest (&argc, argv);
+  } catch (...) {
+    g_warning ("catch 'testing::internal::<unnamed>::ClassUniqueToAlwaysTrue'");
+  }
+
+  gst_init (&argc, &argv);  
+
+  g_warning ("here");
```

Comment:
Maybe this is an unnecessary message for debugging?

Reply:
Thank you.